docs/sources/next/javascript-api/jslib/http-instrumentation-tempo/_index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "http instrument tempo"
+descriptiontion: "k6 tempo instrumentation API"
+weight: 04
+weight: 04
+---
+
+# Tempo
+
+
+With this jslib, you can _instrument_ HTTP requests so that they emit traces as the test runs.
```

Comment:
Would it make sense to add a mention of how this library is related to Grafana Tempo here as well? I think it's only mentioned in the `client.md` page.